### PR TITLE
fix: minor storybook docs and component fixes

### PR DESCRIPTION
**Pull request by pallendes**
This PR adds some minor documentation fixes and components fixes:

- Buttons were not displaying their Icons:
  - Before 
    image
  - Now
    image

- Last Changed Info docs
  - Before
image
  - Now
    image

- Sidesheet had minor issues with the z-index and the spacing between elements
  - Before
    image
  - Now
    image

- Spinner docs: 
  - Before
    image
  - Now
    image

- Tab docs: 
  - Before
    image
  - Now
    image

- Toast docs: 
  - Before
    image
  - Now
    image

- Toggle docs:
  - Before:
    image
  - Now
    image

### CODE REVIEW

File: src/components/sidesheet/sidesheet.tsx [hunk 1]
```diff
@@ -24,11 +24,11 @@ const Sidesheet = ({
         <Transition.Child
           as={Fragment}
           enter="ease-out duration-300"
-          enterFrom="opacity-0"
-          enterTo="opacity-100"
+          enterFrom="opacity-0 z-0"
```
Comment (mnlfischer) [inline src/components/sidesheet/sidesheet.tsx:27]: I also checked the dialog if this clashes but the dialog is implemented as a portal, so the dialog should be always on top.

File: src/components/button/button.tsx [hunk 2]
```diff
@@ -67,12 +67,14 @@ const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
         {loading ? <Spinner size="small" /> : null}
 
         {LeftIcon && !loading ? (
-          <LeftIcon className={iconVariants[type]} />
+          <LeftIcon className={`${iconVariants[type]} w-[1em] h-[1em]`} />
```
Comment (mnlfischer) [inline src/components/button/button.tsx:70]: In our design file the icon for button has a 12px bounding box. So you can use h-3 and w-3 and we are good.

File: src/components/button/button.tsx [hunk 3]
```diff
@@ -67,12 +67,14 @@ const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
...
         ) : null}
 
         {children}
 
-        {RightIcon ? <RightIcon className={iconVariants[type]} /> : null}
+        {RightIcon ? (
+          <RightIcon className={`${iconVariants[type]} w-[1em] h-[1em]`} />
```
Comment (mnlfischer) [inline src/components/button/button.tsx:76]: In our design file the icon for button has a 12px bounding box. So you can use h-3 and w-3 and we are good.

**Review by mnlfischer — CHANGES_REQUESTED** [review]

**Comment by mnlfischer** [comment]
Thanks for polishing. If you address the change we are good to go. Great work!

File: src/components/sidesheet/sidesheet.tsx (hunk 1 above)
Comment (pallendes) [reply src/components/sidesheet/sidesheet.tsx:27]: I see, do we leave it like this to make it look good on Storybook or should I remove it?

**Review by mnlfischer — APPROVED** [review]

**Pull request merged**